### DISCUSSION TITLE
Feature to delete all contacts at once

### DISCUSSION THREAD

**Discussion by solzard** [simple-login/app#878] [Feature Requests]
When you enable or disable "Include sender address in reverse-alias" setting, new contact emails will follow the new setting you made; however, all of existing contacts will not change.

I recently enabled this and asked the SimpleLogin support to delete my bulk contacts instead of me; however they answered that "there is no way to delete all ... at once, and you would need to delete them one by one." 
I read [app/api.md](https://github.com/simple-login/app/blob/master/docs/api.md) and found API calls of `GET aliases`, `GET contacts`, and `DELETE contact` achieve my goal. [This Python script I wrote](https://github.com/solzard/delete_simplelogin_contacts/blob/main/delete_simplelogin_contacts.py) actually works. 

Would you please consider including this feature on the website or on customer support request?

**Comment by G-i-o** [comment]
Absolutely good, sensible request.

I have actually just realized that all receiving email addresses are stored as contact and there's no way to delete them other than deleting them one-by-one (which would involve many, many clicks on the web).

Pity that after three years they haven't taken notice of this yet.
